tests/testthat/test-verb-joins.R

```diff
@@ -76,13 +76,13 @@ test_that("joins with non by variables gives cross join", {
   df1 <- memdb_frame(x = 1:5)
```

Comment:
Test name needs an update maybe should be moved to live next to the other cross join tests?